m4_sessions/automation/session_one/eight_mike_justin_jesse_ethan_dan.md

```diff
@@ -1,3 +1,53 @@
+*   ["Jerry Kaplan: Humans Need Not Apply at Google"](https://www.youtube.com/watch?v=JiiP5ROnzw8) (Google: November 12, 2015)
+
+### Groupings
+  We will group members by opposing stances. Have students sit in a spectrum of how much they like the idea of automation, and from that hand select one each from opposing sides and a middle. This should facilitate the most conversation and opposing viewpoints.
+  
+ ## Session Outline
+
+| Timing        | Facilitator Talking Points           | Participant/Facilitator Actions  |
+| ------------- | ------------- | ----- |
+| 5 min | Introduction: Welcome participants.  Introduce yourself and your role.  Review the outcomes for the session and the connection to the greater Turing mission.  Ask students about survey, and read out some responses.
+| 5 min | Warm Up: Kick off an activity where students will organize in the room along a spectrum of their attitudes toward automation. Describe one end as representing people who feel strongly that automation is, as Elon Musk says, "summoning a demon." On the other end, people who feel automation is going to take humanity to new, positive, exciting potential. | Facilitator will instruct students to line up in between these poles as appropriate. At facilitator's prompting, students should line up along the spectrum of attitudes toward automation. This arrangement will be used to select dynamic groups |
+| 2 min | Introduce video and reflection questions to students. <br> 1) How serious an impact do you think automation will have on our economy from now until you retire? <br> 2) Where do you think you fall across the spectrum of people that will be affected by these changes? <br> 3) What do you think it means to be a responsible citizen around this issue? <br> 4) What role can Turing play in advancing civic responsibility in response to the challenges automation will bring?     
+| 0 min | ----------|.
+| 15 mins | One of these videos will be shown, to be selected after quick meeting before session. <br> 1. Kick off ["Humans Need Not Apply"](https://www.youtube.com/watch?v=7Pq-S557XQU) video. <br> 2. Kick off ["Do labor-saving robots spell doom for American workers?"](http://www.pbs.org/newshour/bb/labor-saving-robots-spell-doom-american-workers/) video | Students should take notes during the video.  Ask them to put away cell phones/computers if needed.
+| 13 min | Introduce discussion section: Students should identify a timekeeper, a faciliator, and a notetaker.  Each student 
+should spend about 2 minutes sharing what surfaced during their writing. After individual sharing, the group facilitator can help the group decide what themes from people's writing to discuss in more depth. | Facilitator: walk the room and listen to group discussions.  Identify a few key points or themes from the small group discussions that you might want to share with the whole group |
+
+### A second video will be shown to close out the session.
```

Comment:
@Sh1pley is taking point on finding the second video. Let us know if you have additional questions, thanks!